src/bundle/ui-dev/src/modules/content-tree/services/content.tree.service.js

```diff
@@ -28,8 +28,13 @@ export const loadLocationItems = ({ siteaccess }, parentLocationId, callback, li
         .catch(showErrorNotification);
 };
 
-export const loadSubtree = ({ token, siteaccess }, subtree, callback) => {
-    const request = new Request(`${ENDPOINT_LOAD_SUBTREE}`, {
+export const loadSubtree = ({ restInfo: { token, siteaccess }, subtree, sortClause, sortOrder }, callback) => {
```

Comment:
```suggestion
export const loadSubtree = ({ token, siteaccess, subtree, sortClause, sortOrder }, callback) => {
```

Do not pass `restInfo` object just pass the `token` and `siteaccess` this is more consistent with other places.